ci/baremetal/baremetal-cluster.lokocfg.envsubst

```diff
@@ -31,4 +31,9 @@ cluster "bare-metal" {
     "node2",
     "node3",
   ]
+
+  labels = [
+    "testing.io=yes",
+    "roleofnode=testing",
+  ]
```

Comment:
Squash this with enabling the test

Reply:
Actually, I'm not sure if I should squash.